.sass-lint.yml

```diff
@@ -0,0 +1,182 @@
+# sass-lint config generated by make-sass-lint-config v0.1.2
+#
+# The following scss-lint Linters are not yet supported by sass-lint:
+# DisableLinterReason, ElsePlacement, PropertyCount, SelectorDepth
+# SpaceAroundOperator, TrailingWhitespace, UnnecessaryParentReference, Compass::*
```

Comment:
I haven't changed any of the config that I pulled, for CSS/Sass I'm not as particular as I am in JavaScript. As long as the Sass is kept to some standard I'm pretty happy.